vars/checkoutQaInternal.groovy

```diff
@@ -2,8 +2,8 @@
 
 def call(Map params = [:]) {
     dir("scylla-qa-internal") {
-        git(url: 'git@github.com:scylladb/scylla-qa-internal.git',
+        git(url: 'git@github.com:vponomaryov/scylla-qa-internal.git',
```

Comment:
please review this soe we can merge this one

Reply:
I don't understand your request. Too many typos

Reply:
Removed changes for this file assuming that the required changes will be merged in that repo.